src/mixxxapplication.cpp

```diff
@@ -199,6 +200,11 @@ bool MixxxApplication::notify(QObject* pTarget, QEvent* pEvent) {
 
     bool ret = QApplication::notify(pTarget, pEvent);
 
+    VERIFY_OR_DEBUG_ASSERT(pTarget != nullptr) {
+        qWarning() << "Processed" << pEvent->type() << "for null pointer, this is probably a bug!";
```

Comment:
This warning is redundant because QT also logs one: 
https://github.com/qt/qtbase/blob/011081998703569a1dfcd17edcb65b27e1f9a8f9/src/widgets/kernel/qapplication.cpp#L2577